### PR TITLE
LSIF: Update UI to reflect change from LSIF jobs to uploads

### PR DESCRIPTION
**Pull request by efritz**
Update the UI to reflect changes from https://github.com/sourcegraph/sourcegraph/pull/6875. This PR replaces all LSIF job resources with LSIF upload resources, which are similar but a bit more specific. There is no job type and the arguments are fixed (repo, commit, root, and filename).

Both internal jobs and scheduled jobs are no longer a concept and have been removed from the UI.

This will be merge **only after** https://github.com/sourcegraph/sourcegraph/pull/6875.

### CODE REVIEW

File: web/src/enterprise/site-admin/SiteAdminLsifUploadPage.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        params: { id },
+    },
+}) => {
+    useEffect(() => eventLogger.logViewEvent('SiteAdminLsifUpload'))
+
+    const uploadOrError = useObservable(
+        useMemo(() => fetchLsifUpload({ id }).pipe(catchError((error): [ErrorLike] => [asError(error)])), [id])
+    )
+
+    console.log(uploadOrError)
```
Comment (lguychard) [inline web/src/enterprise/site-admin/SiteAdminLsifUploadPage.tsx]: Unintentional?

```suggestion
```

File: web/src/enterprise/site-admin/SiteAdminLsifUploadsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    public componentDidMount(): void {
+        eventLogger.logViewEvent('SiteAdminLsifUploads')
+
+        this.subscriptions.add(
+            this.updates
+                // Do not set statsOrError as null here to indicate loading
+                // so that the stats don't disappear during navigation, which
+                // causes some weird jitter.
+                .pipe(switchMap(() => fetchLsifUploadStatistics().pipe(catchError(err => [asError(err)]))))
+                .subscribe(stats => this.setState({ statsOrError: stats }))
```
Comment (lguychard) [inline web/src/enterprise/site-admin/SiteAdminLsifUploadsPage.tsx]: Nit

```suggestion
                .subscribe(statsOrError => this.setState({ statsOrError }))
```

File: web/src/enterprise/site-admin/SiteAdminLsifUploadPage.tsx (hunk 1 above)
Comment (efritz) [reply web/src/enterprise/site-admin/SiteAdminLsifUploadPage.tsx]: Yup!

**Comment by unknwon** [comment]
Sorry for being late! looks like all just renames from "job" to "upload", LGTM.

**Pull request merged**